### ISSUE TITLE
Player.addItemInventory() possible issue

### ISSUE THREAD

**Issue by Black-DeathPE** [zhuowei/MCPELauncher#1031]
One of my mods have a system which takes an item from the inventory (id, -1) and adds another item in place (newid, 1). The icon on the hotbar does not update and also this system seems to crash after a certain amount of times of it being called. I know this isn't my mod's issue, since I haven't touched the code since 0.14.2 and it wasn't an issue then.

**Comment by Black-DeathPE** [comment]
Additionally, when I first add items no icons appear in the hotbar. The only way to update the icons is by opening the inventory.

**Comment by Black-DeathPE** [comment]
When items are just added it updates icons about 1/2 the time.

**Comment by Aternix** [comment]
Yep... adding custom items in creative is possible but Zhuowei must fix the errors first.
